.circleci/config.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     # Pull docker image from docker hub
     # XTERM used for better catkin_make output
     docker:
-      - image: usdotfhwastol/carma-base:3.7.1
+      - image: usdotfhwastoldev/carma-base:develop
```

Comment:
Are we going to replace the carma base version to develop version when we move to release? if not, maybe leave this unchanged. If we have an updated release carma base version, please update with the new release version.

Reply:
Normally for a release we do want to target the candidate image so this should be 
usdotfhwastolcandidate/carma-base:vanden-plas

Reply:
@msmcconnell if I remembered correctly JB would just make the changes against the vanden plas branch directly.  I would rather keep this one clean merge if possible. But  let's see first if Kyle has any comments below. If it's candidate specific, then separate PR should be made against the candidate branch, since this just represents what's been on develop all along.

Reply:
It is candidate specific, so does probably make sense to be a seperate branch and pr